File: python/pallycon/config/fairplay_hdcp_enforcement.py
FAIRPLAY_HDCP_ENFORCEMENT = (-1, 0, 1)

HDCP_NONE = FAIRPLAY_HDCP_ENFORCEMENT[0]  # 'HDCP_NONE'
HDCP_TYPE_0 = FAIRPLAY_HDCP_ENFORCEMENT[1]  # 'HDCP_TYPE_0'
HDCP_TYPE_2 = FAIRPLAY_HDCP_ENFORCEMENT[2]  # 'HDCP_TYPE_2',


def check(hdcp_enforcement):
    return hdcp_enforcement in FAIRPLAY_HDCP_ENFORCEMENT

File: java/src/main/java/com/pallycon/sample/token/policy/securityPolicy/fairplay/FairplayHdcpEnforcement.java
package com.pallycon.sample.token.policy.securityPolicy.fairplay;

/**
 * for @security_policy @fairplay
 */
public enum FairplayHdcpEnforcement {

    HDCP_NONE("HDCP_NONE", -1),
    HDCP_TYPE_0("HDCP_TYPE_0", 0),
    HDCP_TYPE_1("HDCP_TYPE_1", 1);

    private String type;
    private int value;

    FairplayHdcpEnforcement(String type, int value) {
        this.type = type;
        this.value = value;
    }

    public String getType() {
        return type;
    }

    public int getValue() {
        return value;
    }
}
